src/Module.php

```diff
@@ -378,6 +383,13 @@ public function renderToolbar($event)
         $view = $event->sender;
         echo $view->renderDynamic('return Yii::$app->getModule("' . $this->id . '")->getToolbarHtml();');
 
+        if (!empty($this->skipAjaxRequestUrl)) {
+            foreach ($this->skipAjaxRequestUrl as $key => $route) {
+                $this->skipAjaxRequestUrl[$key] = Url::to([$route]);
+            }
+            echo '<script> var skipAjaxRequestUrl =' . json_encode($this->skipAjaxRequestUrl) . '</script>';
```

Comment:
I think it is better to pass these via HTML data attribute instead of relying on global variables.

Reply:
OK, I'll rewrite it